src/commands/Configuration/configure/automod.ts

```diff
@@ -1,31 +1,54 @@
 
-export default class extends SubCommand {
-	constructor(client: BulbBotClient, parent: Command) {
-		super(client, parent, {
-			name: "automod",
-			description: "Configure the automod in your server",
-			category: "Configuration",
-			subCommands: [enable, settings, add, disable, remove, punishment, limit],
-			aliases: ["am"],
-			usage: "<action>",
-			examples: ["configure automod enable", "configure automod edit", "configure automod punishment"],
-			argList: ["action:String"],
-			minArgs: 1,
-			maxArgs: -1,
-			clearance: 75,
-			userPerms: ["MANAGE_GUILD"],
-			clientPerms: ["EMBED_LINKS"],
-		});
-	}
+	const selectRow = new MessageActionRow().addComponents(
+		new MessageSelectMenu()
+			.setCustomId("automod")
+			.setPlaceholder(await client.bulbutils.translate("config_main_placeholder", interaction.guild?.id, {}))
+			.setOptions([
+				{ label: "Add/remove", value: "add_remove", description: await client.bulbutils.translate("config_automod_main_add_remove_description", interaction.guild?.id, {}) },
+				{ label: "Enable/Disable", value: "enable_disable", description: await client.bulbutils.translate("config_automod_main_enable_disable_description", interaction.guild?.id, {}) },
+				{ label: "Limit", value: "limit", description: await client.bulbutils.translate("config_automod_main_limit_description", interaction.guild?.id, {}) },
+				{ label: "Punishment", value: "punishment", description: await client.bulbutils.translate("config_automod_main_punishment_description", interaction.guild?.id, {}) },
+				{ label: "Overview", value: "overview", description: await client.bulbutils.translate("config_automod_main_overview_description", interaction.guild?.id, {}) },
+			]),
+	);
+
+	const backRow = new MessageActionRow().addComponents([new MessageButton().setCustomId("back").setLabel(back).setStyle("DANGER")]);
+
+	interaction.deferred
+		? await interaction.editReply({ content: header, components: [selectRow, backRow] })
+		: await interaction.update({ content: header, components: [selectRow, backRow], embeds: [] });
+
+	const filter = i => i.user.id === interaction.user.id;
+	const collector = interaction.channel?.createMessageComponentCollector({ filter, max: 1, time: 60000 });
+
+	collector?.on("collect", async (i: MessageComponentInteraction) => {
+		if (i.isButton()) {
+			await require("./main").default(i, client);
+		} else if (i.isSelectMenu()) {
+			switch (i.values[0]) {
+				case "add_remove":
+					await require("./automod/add").default(i, client);
+					break;
+				case "enable_disable":
+					await require("./automod/enable").default(i, client);
+					break;
+				case "limit":
+					await require("./automod/limit").default(i, client);
+					break;
+				case "punishment":
+					await require("./automod/punishment").default(i, client);
+					break;
+				case "overview":
+					await require("./automod/overview").default(i, client);
+					break;
+			}
```

Comment:
```suggestion
			const resolvedValue = i.values[0].split('_')[0];
			await require(`./automod/${resolvedValue}`).default(i, client);
```
`resolvedValue` will return the first word from the received string, then just require the file with it.